src/bundle/ui-dev/src/modules/multi-file-upload/multi.file.upload.module.js

```diff
@@ -97,6 +97,9 @@ export default class MultiFileUploadModule extends Component {
     }
 
     showUploadPopup() {
+        this.configRootNode.removeEventListener('drop', this.handleDropOnWindow, false);
+        this.configRootNode.removeEventListener('dragover', this.preventDefaultAction, false);
+
         this.setState((state) => ({ ...state, popupVisible: true, itemsToUpload: [] }));
```

Comment:
```suggestion
        this.setState((state) => ({ ...state, popupVisible: true, itemsToUpload: [], allowDropOnWindow: false }));
```
I think this should do the trick. If you remove listeners on modal opening the flow when user opens a modal then close it and want drop some file will break.